examples/simulation/rutherford/src/FairRutherfordGeo.cxx

```diff
@@ -1,36 +1,16 @@
 /********************************************************************************
- * Copyright (C) 2014-2023 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
+ * Copyright (C) 2014-2024 GSI Helmholtzzentrum fuer Schwerionenforschung GmbH  *
  *                                                                              *
  *              This software is distributed under the terms of the             *
  *              GNU Lesser General Public Licence (LGPL) version 3,             *
  *                  copied verbatim in the file "LICENSE"                       *
  ********************************************************************************/
 #include "FairRutherfordGeo.h"
 
-#include <cstdio>   // for snprintf
-
 FairRutherfordGeo::FairRutherfordGeo()
     : FairGeoSet()
 {
     // Constructor
-    fName = "rutherford";
-    maxModules = 10;
-}
-
-const char* FairRutherfordGeo::getModuleName(Int_t m)
-{
-    /** Returns the module name of FairRutherford number m
-      Setting MyDet here means that all modules names in the
-      ASCII file should start with FairRutherford otherwise they will
-      not be constructed
-  */
-    std::snprintf(modName, sizeof(modName), "rutherford%i", m + 1);
-    return modName;
-}
-
-const char* FairRutherfordGeo::getEleName(Int_t m)
-{
-    /** Returns the element name of Det number m */
-    std::snprintf(eleName, sizeof(eleName), "rutherford%i", m + 1);
-    return eleName;
+    SetName("rutherford");
```

Comment:
Instead of `SetName` in the ctor, what about a new ctor for FairGeoSet that takes a `const char*` and calls the `TNamed(name, "")` ctor?

Reply:
Maybe even a `FairGeoSet(const char*, unsigned int maxmodules = 0)`?

Reply:
I agree but as said previously the main topi of this PR is to remove the usage of sprintf, so I would like to not change this now but with additional PRs.